public/js/till.js

```diff
@@ -0,0 +1,76 @@
+function Till(details){
+  this.details = details;
+  this.taxes = 0;
+  this.total = 0;
```

Comment:
The size of this constructor is an immediate code smell that this object is doing too much stuff